charts/loki-distributed/README.md

```diff
@@ -113,10 +114,9 @@ kubectl delete statefulset RELEASE_NAME-loki-distributed-querier -n LOKI_NAMESPA
 | gateway.image.registry | string | `"docker.io"` | The Docker registry for the gateway image |
 | gateway.image.repository | string | `"nginxinc/nginx-unprivileged"` | The gateway image repository |
 | gateway.image.tag | string | `"1.19-alpine"` | The gateway image tag |
-| gateway.ingress.annotations | object | `{}` | Annotations for the gateway ingress |
+| gateway.ingress.annotations | object | `{}` | Ingress Class Name. MAY be required for Kubernetes versions >= 1.18 ingressClassName: nginx -- Annotations for the gateway ingress |
 | gateway.ingress.enabled | bool | `false` | Specifies whether an ingress for the gateway should be created |
-| gateway.ingress.hosts | list | `[{"host":"gateway.loki.example.com","paths":[{"path": "/"}]}]` | Hosts configuration for the gateway ingress |
-| gateway.ingress.ingressClassName | string | `nil` | Ingress Class Name. MAY be required for Kubernetes versions >= 1.18 |
```

Comment:
@AndreZiviani It looks like something went wrong with the doc generation with helm-docs. Could you please double check this?

Reply:
I'm not sure what you mean, if you are talking specifically about `ingressClassName` it was commented out on #522 but looks like the docs wasn't updated

Reply:
Yeah it seems you are right.